<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="5,1,2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="DrawableView" name="view"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,0,0">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QPushButton" name="btn_add_box">
              <property name="text">
               <string>add box</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_revise_box">
              <property name="text">
               <string>revise box</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QPushButton" name="btn_add_mask">
              <property name="text">
               <string>add mask</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_revise_mask">
              <property name="text">
               <string>revise mask</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QPushButton" name="btn_mode">
            <property name="text">
             <string>…</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_ok">
            <property name="text">
             <string>❤</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_delete_obj">
            <property name="text">
             <string>✘</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="lw_objs"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1,0,0,1,0,0,0,0,0,0">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>object annotations</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tw_obj_message"/>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>category labels</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="lw_labels"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="btn_add_category">
          <property name="text">
           <string>add category</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_delete_category">
          <property name="text">
           <string>delete category</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>prompt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_prompt"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>default category: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_default_category">
          <property name="text">
           <string>object</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="btn_open_pic">
            <property name="text">
             <string>open</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_save_pic">
            <property name="text">
             <string>save</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="btn_detect_box">
            <property name="text">
             <string>detect box</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_detect_mask">
            <property name="text">
             <string>detect mask</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1086</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrawableView</class>
   <extends>QGraphicsView</extends>
   <header>ui.canvas_ui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
